backend/app/adapter/sqldb/user_short_link.go

```diff
@@ -63,6 +63,24 @@ func (u UserShortLinkSQL) FindAliasesByUser(user entity.User) ([]string, error)
 	return aliases, nil
 }
 
+// IsAliasOwnedByUser verifies if the ShortLink alias belongs to the given user.
+func (u UserShortLinkSQL) IsAliasOwnedByUser(user entity.User, alias string) (bool, error) {
```

Comment:
```suggestion
func (u UserShortLinkSQL) HasMapping(user entity.User, alias string) (bool, error) {
```

Reply:
I prefer delegating the concept of ownership to use case.

Reply:
We may have different algorithm to determine `Ownership` in the future. This leaves us the flexibility to make changes.